File: example/whisper/example-javascript.js
const fetch = require('node-fetch');
const fs = require('fs');
const FormData = require('form-data');

const url = "https://api.regolo.ai/v1/models/whisper-large-v3/transcriptions";
const apiKey = "$REGOLOAI_API_KEY";

const form = new FormData();
form.append("model", "whisper-1");
form.append("file", fs.createReadStream("file.mp3"));

fetch(url, {
	  method: 'POST',
	  headers: { "Authorization": `Bearer ${apiKey}`, ...form.getHeaders() },
	  body: form
})
  .then(res => res.json())
  .then(data => console.log(data))
  .catch(err => console.error('Error:', err));

File: example/stablediffusion/example-javascript.py
const fetch = require('node-fetch');

const url = process.env.ENDPOINT;
const token = process.env.REGOLO_TOKEN;

const data = { data: ["Cat playing the piano"] };

fetch(url, {
        method: 'POST',
            headers: {
                        'Content-Type': 'application/json',
                                'Authorization': `Bearer ${token}`
                                    },
                body: JSON.stringify(data)
                })
.then(response => response.json())
.then(data => console.log(data))
.catch(error => console.error('Error:', error));
